fmlloader/src/main/java/net/minecraftforge/fml/loading/moddiscovery/AbstractModProvider.java

```diff
@@ -35,6 +36,7 @@
 public abstract class AbstractModProvider implements IModProvider {
     private static final   Logger LOGGER    = LogUtils.getLogger();
     protected static final String MODS_TOML = "META-INF/mods.toml";
+    private static final String MODULE_INFO = "module-info.class";
```

Comment:
Should maybe be protected for third-parties that extend AbstractModProvider?
```suggestion
    protected static final String MODULE_INFO = "module-info.class";
```

Reply:
I mean it doesn't hurt, but there is also no reason to. This has nothing to do with providing/locating mods and more with parsing Jar. Honestly I would stick it in Jar itself but I don't feel like editing ModLauncher right now.